pkg/descriptions.json

```diff
@@ -733,6 +733,19 @@
       "System.Drawing.Icon"
     ]
   },
+  {
+    "Name": "System.Drawing.Extensions",
```

Comment:
Not really, just because I suck at naming and I thought this should've been supported by netfx and in netfx System.Drawing.Design already exists, but I guess we should be fine now right? Any suggestions?

Reply:
>  I suck at naming 

[Welcome to the club.](https://quotesondesign.com/phil-karlton/)

> in netfx System.Drawing.Design already exists, but I guess we should be fine now right?

No we can't use that name, unfortunately.  These were in System.Drawing in desktop, so if we moved them upstack in Core it would create a cycle among the shims.  Moreover we can only use 1:1 names in .netcore if we fully satisfy the API (or at least to the extent that we implement it). It's quite possible S.D.Design will exist in some form in the designer and a shim will need to point to all of the new locations for those types.

The distinction between these types and those in System.Drawing.Design is that these are the "glue" to for runtime types to support the designer without referencing the designer itself, whereas System.Drawing.Design contains the actual designer components: `System.Drawing.Design.Primitives`?

I hate to fall back to our go-to Primitives/Extensions/Common names, but heck, at least it's not *Core*.

Reply:
> I hate to fall back to our go-to Primitives/Extensions/Common names, but heck, at least it's not Core.

haha I know. `System.Drawing.Design.Primitives` sounds good to me.

Reply:
Are the types/apis being added a pure subset of what lives in `System.Drawing.Design`? If so I'm good with the name `System.Drawing.Design.Primitives` but if not and it is more of a grab-bag of types then I prefer something like `System.Drawing.Extensions` assuming they are all related to drawing.

Reply:
I would assert that the purpose of this assembly is to represent the things in the System.Drawing.Design namespace (or related to the "design time" scenarios) that are relevant for *runtime* libraries.  Anything else in the System.Drawing.Design namespace that's needed to implement the designers should go in a different library.  Anything else that is not in System.Drawing.Design but related to Drawing should go in a different library (Primitives or Common).